modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/comments/DevPortalCommentTest.java

```diff
@@ -77,6 +85,26 @@ public DevPortalCommentTest(TestUserMode userMode) {
     @BeforeClass(alwaysRun = true)
     public void setEnvironment() throws Exception {
         super.init(userMode);
+        User non_admin_user;
+        String non_admin_username;
```

Comment:
Shall we avoid "_" from usernames. Please check and correct this in all the places